tests/framework/db/CommandTest.php

```diff
@@ -314,7 +314,11 @@ public function testInsertExpression()
     public function testCreateTable()
     {
         $db = $this->getConnection();
-        $db->createCommand("DROP TABLE IF EXISTS testCreateTable;")->execute();
+
+        try {
+            $db->createCommand()->dropTable('testCreateTable')->execute();
+        } catch (\Exception $ex) {
```

Comment:
In case of empty `catch` it's good to add comment about the fact that it's on purpose.